Blizzard.IO.RabbitMQ.Builders/NetqRabbitPublisherBuilder.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Blizzard.IO.Core;
+using Blizzard.IO.RabbitMQ.Entities;
+using Blizzard.IO.RabbitMQ.Extensions;
+using Blizzard.IO.Serialization.Json;
+using EasyNetQ;
+using Microsoft.Extensions.Logging;
+
+namespace Blizzard.IO.RabbitMQ.Builders
+{
+    public class NetqRabbitPublisherBuilder<TData> : BaseNetqRabbitBuilder
+    {
+        private RabbitExchange _destinationExchange = null;
+        private ISerializer<TData> _serializer = new JsonSerializer<TData>();
+        private string _routingKey = "/";
+        private bool _isAbstract;
+        private IConverter<RabbitMessageProperties, MessageProperties> _converter;
+        private ILoggerFactory _loggerFactory;
```

Comment:
Why do you have your own logger factory where the base has another logger factory?

Reply:
f